plugins/modules/df_service.py

```diff
@@ -78,12 +79,44 @@
     required: False
     aliases:
       - use_public_load_balancer
-  ip_ranges:
+  loadbalancer_ip_ranges:
+    description: The IP ranges authorized to connect to the load balancer
+    type: list
+    required: False
+  kube_ip_ranges:
     description: The IP ranges authorized to connect to the Kubernetes API server
     type: list
     required: False
-    aliases:
-      - authorized_ip_ranges
+  cluster_subnets:
+    description:
+      - Subnet ids that will be assigned to the Kubernetes cluster
+      - Mutually exclusive with the cluster_subnets_filter option 
+    type: list
+    required: False
+  cluster_subnets_filter:
+    description:
+      - JMESPath expression to filter the subnets to be used for the Kubernetes cluster
```

Comment:
Add a link to the docs:

"L(JMESPath,https://jmespath.org/) expression..."

Reply:
Done